srcalc/src/main/webapp/WEB-INF/views/tiles/enterVariables.jsp

```diff
@@ -99,17 +105,17 @@
         <script type="text/javascript" src="${dataTablesUrl}"></script>
         <script type="text/javascript">
         $(document).ready(function(){
-        	initEnterVariablesPage();
         	// Set up the properties for the procedures DataTable
-	    	var table = $("#procedureTable").dataTable( {
-	    		// Make the radio button column smaller so that IE
-	    		// will adjust column widths properly.
+	    	$("#procedureTable").dataTable({
+	    		"data": procedureArray,
 	    		"aoColumnDefs":[
```

Comment:
"aoColumnDefs" and the other prefixed option names are deprecated in DataTables 1.10.